spec/converterbase_spec.rb

```diff
@@ -293,6 +293,105 @@ def to_ruby(match, m1, m2, open_close)
                "追撃をしない俺に振り返り、｜再度《パワースラッシュ 》"
       end
     end
+
+    describe "｜を使ったルビ位置の指定" do
+      it "《 》基本" do
+        expect(to_ruby( \
+               "｜山田太郎《やまだたろう》",
+               "｜山田太郎", "やまだたろう", ["《", "》"])).to eq \
```

Comment:
__[rubocop]__
* Layout/MultilineMethodCallBraceLayout: Closing method call brace must be on the line after the last argument when opening brace is on a separate line from the first argument.

<!-- Comment by [SideCI](https://sideci.com) -->